google/gax/__init__.py

```diff
@@ -134,7 +139,8 @@ def merge(self, options):
             return CallSettings(
                 timeout=timeout, retry=retry,
                 page_descriptor=self.page_descriptor, page_token=page_token,
-                bundler=bundler, bundle_descriptor=self.bundle_descriptor)
+                bundler=bundler, bundle_descriptor=self.bundle_descriptor,
+                kwargs=self.kwargs)
```

Comment:
Let's a actually do

```
if options.kwargs == OPTION_INHERIT:
  kwargs = self.kwargs
else:
  kwargs = self.kwargs.copy()
  kwargs.update(options.kwargs)
```

I can't tell where `options.kwargs` is currently being used -- maybe we are just ignoring it at the moment?